guides/getting-started.md

```diff
@@ -2,14 +2,14 @@
 
 This guide explains how to setup your Angular project to begin using Angular Material. It includes information on prerequisites, installing Angular Material, and optionally displaying a sample material component in your application to verify your setup.
 
-### New to Angular ?
+*Angular Resources*
 
 If you are new to Angular or getting started with a new Angular application, see [Angular's full Getting Started Guide](https://angular.io/start) and [Setting up your environment](https://angular.io/guide/setup-local).
 
-For existing applications, follow the steps below to begin using Angular Material:
+### Follow these steps to begin using Angular Material
 
-
-### Step 1: Install Angular Material npm packages
+#### Step 1
+**Install Angular Material NPM packages**
```

Comment:
Wouldn't the previous approach be better for SEO? Moving it out means that we don't have any keywords in the header anymore. Also changing the anchors like this will mess up any places where we've linked to it until now.

Reply:
I would prefer to keep this text in the header (here and below)

Reply:
I don't think that we've linked to these yet because they were changed in PR https://github.com/angular/components/pull/16654/files and those changes have never been published to the docs site yet.

It does help to have those words in the headers for SEO, but it makes for really long anchors (see description).

Can we shorten this to just?
> ### Install Angular Material

Which would be `/guide/getting-started#install-angular-material`.

Does the "step 1" and "step 2" text add any value? Can we remove that?

Reply:
Updated, PTAL